src/plugins/data_table/filter_view.js

```diff
@@ -56,7 +56,15 @@ var DataTableFilter = (function(options) {
 
   var StringFilterMenu = DataTableFilterMenu.extend({
     menuTemplate: _.template('\
-        <input class="filter-string" id ="value" type="text" placeholder="Search <%= title %>" />\
+      <div class="filterMenu filterMenu-string">\
+        <div class="filter-text">Show:</div>\
+        <div class="icon-addon addon-sm">\
+          <input class="filter-string form-control input-sm" id="value" type="text" name="filter-string" />\
```

Comment:
can you remove `id="value"` please? i know existed before but there will be lots of these on the page, and w/o a unique ID